docker/ansible/systemd.yml

```diff
@@ -0,0 +1,24 @@
+# Playbook to install the systemd service and timers files
+
+- hosts: localhost
+  become: yes
+  vars:
+    # Required to use docker_compose properly
+    ansible_python_interpreter: /usr/bin/python3
+  tasks:
+    - name: Copy systemd service
+      copy:
+        src: /home/viame/docker/ansible/viame-deploy.service
+        dest: /etc/systemd/system/viame-deploy.service
+
+    - name: Copy systemd timer
+      copy:
+        src: /home/viame/docker/ansible/viame-deploy.timer
+        dest: /etc/systemd/system/viame-deploy.timer
+
+    - name: Enable timer
+      systemd:
+          name: viame-deploy.timer
+          state: started
```

Comment:
I haven't tested this, but are you certain this will cause changes in `viame-deploy.timer` to be picked up?

I see the `daemon-reload`, but since `state: started` doesn't cause the timer to bounce, I'm not sure if the changes in the timer file are picked up.  is `state: restarted` safer?   For example, if the schedule changes, but ansible doesn't actually restart the timer, do the schedule changes get noticed?

Reply:
I've tested this and I believe what's there has the desired effect you're describing, but I can double check and change that to `restarted` to be safe.

Reply:
96e5804